orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/knox.settings.j2

```diff
@@ -2,4 +2,5 @@
 setsettings PROXYUSER_KNOX_GROUPS *
 setsettings PROXYUSER_KNOX_HOSTS *
 setsettings PROXYUSER_KNOX_PRINCIPAL knox
-setsettings PROXYUSER_KNOX_USERS *
\ No newline at end of file
+setsettings PROXYUSER_KNOX_USERS *
+setsettings FRONTEND_URL https://{{ knox_address }}:8443/
```

Comment:
I have NOT tested this with a deployment (not sure how), but would be good to check that the following two URLs work with CM and Knox:

* `https://KNOX_HOST:8443/CLUSTER_NAME/cdp-proxy/cmf/home`
* `https://KNOX_HOST:8443/cmf/home`

The first one in the list is what is generated in the CB gateway tab so want to make sure that still works. The second one is where CM is really rooted once any CM links are clicked on.

Reply:
As far as I know this property is not used in CM yet, but yes we can test if the second works also.

Reply:
Let me see if I can track that change down in the next few min.

Reply:
Hmmm so yea the CM change isn't in yet. Sorry jumped the gun worrying about it.